features/org.wso2.carbon.identity.sso.saml.server.feature/resources/samlsso_response.html

```diff
@@ -1,20 +1,104 @@
+<!DOCTYPE html>
+
+<!--
+~    Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
+~
+~    This software is the property of WSO2 Inc. and its suppliers, if any.
+~    Dissemination of any information or reproduction of any material contained
+~    herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+~    You may not alter or remove any copyright or other notice from copies of this content."
+-->
+
 <!-- 
 Variables $acUrl, $response, $relayState and $additionalParams will be replaced by the corrosponding values  
 -->
+
 <html>
-	<body>
-        	<p>You are now redirected back to $acUrl 
-        	If the redirection fails, please click the post button.</p>
 
-        	<form method='post' action='$acUrl'>
-       			<p>
-					<!--$params-->
+<head>
+    <title>Redirecting</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
why do we rely on a CDN for this stylesheet?

Reply:
This is a single HTML file, we're not able to import external files